packages/dvilela/connections/mirror_db/connection.yaml

```diff
@@ -16,7 +16,7 @@ protocols:
 - valory/srr:0.1.0:bafybeihrixgptxuqpf2s4nujypwlv5aed2nboozq5kl4c26cxw6r7si2x4
 class_name: MirrorDBConnection
 config:
-  mirror_db_base_url: ${MIRROR_DB_BASE_URL}
+  mirror_db_base_url: ${MIRROR_DB_BASE_URL:-https://afmdb.autonolas.tech}
```

Comment:
This is not correct. Please check the twikit connection and do it as we do it there